### ISSUE TITLE
Allow for custom alsa mixer name and device

### ISSUE THREAD

**Issue by vvv-ca** [jlonardi/node-audio-linux#1]
Any chance you could add the ability to specify the name of the device and mixer to use?
My audio card does not expose a Master mixer.

Thanks!

**Comment by jlonardi** [comment]
Yes, this is a reasonable thing to add. I'll take a look into it.
